packages/ibm-products/src/components/BasicHeader/index.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright IBM Corp. 2020, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export { CreateFullPageHeader } from './BasicHeader';
```

Comment:
```suggestion
export { CreateFullPageHeader } from './BasicHeader';
```

Should probably update this to be consistent with whatever naming we end up using (basic/simple).

Reply:
Done.